apps/api/src/app/environments/environment.ts

```diff
@@ -100,6 +100,11 @@ const devConfig = {
       process.env.XROAD_TJODSKRA_API_PATH ?? '/SKRA-Protected/Einstaklingar-v1',
     clientId: process.env.XROAD_CLIENT_ID ?? 'DEV/GOV/10000/island-is-client',
   },
+  paymentDomain: {
+    username: 'isl_aranja_p',
+    password: 'vogur.123',
+    url: 'https://tbrws-s.hysing.is',
```

Comment:
is hysing fine with us exposing these secrets to the world? nb this should be considered compromised now if not

Reply:
@eirikurn

Reply:
We are getting a new password, along with adding the user/pass to local secrets to prevent further compromising.